src/components/Modals/FiatRamps/views/FiatForm.tsx

```diff
@@ -141,7 +156,7 @@ export const FiatForm: React.FC<FiatFormProps> = ({
 
   return (
     <Overview
-      assetId={selectedAssetId ?? assetId}
+      assetId={selectedAssetId ?? assetId ?? defaultAsset?.assetId ?? ethAssetId}
```

Comment:
The last resort `ethAssetId` is here for paranoia, but should never eventuate unless logic upstream has failed.

Reply:
Do we want to return null instead if `!(selectedAssetId || assetId || defaultAsset?.assetId)`?
Then we can remove the latest nullish coalescing and non-null assert the whole expression